decidim-design/app/views/layouts/decidim/design/application.html.erb

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Decidim Design Guide</title>
```

Comment:
It's important to add unique titles for every page. Most of the time, it should be the same as the heading (`h1`). 
In Decidim we use this helper, but you may need to include it in the controller first:

```ruby
<% add_decidim_page_title("Color") %>     
```

And also make aware this layout that needs to consume there with 

```ruby
 <title><%= decidim_page_title %></title>
``` 

You need to add the ` add_decidim_page_title` in all the pages. 

This needs to be done in this PR, as this problem is something that if it isn't done now, then it keeps being more effort in the future.

Reply:
Added. I've used the content provided by the `:heading` content_for, so it's expected from all pages to provide a unique value for it